src/applications/personalization/profile/components/direct-deposit/BankInfoEDU.jsx

```diff
@@ -217,14 +217,17 @@ export const DirectDepositEDU = ({
     return notEligibleContent;
   };
 
-  const directDepositData = [
-    // the table can show multiple states so we set its value with the
-    // getBankInfo() helper
-    {
-      title: 'Account',
```

Comment:
With this change, we only add the `title` prop to the table data if the user has direct deposit set up